libs/langgraph/langgraph/constants.py

```diff
@@ -22,6 +23,7 @@
 END = "__end__"
 
 CHECKPOINT_NAMESPACE_SEPARATOR = "|"
+SEND_CHECKPOINT_NAMESPACE_SEPARATOR = ":"
```

Comment:
why is this a different separator?

Reply:
so that we can distinguish between node names and IDs, otherwise we don't know if the last item in the `checkpoint_ns` is a node name or an ID